components/html/modules/content-newsletter-text-only/_landing.html

```diff
@@ -1,18 +1,20 @@
 <div class="row">
 	<section class="intro col-xs-12 col-sm-12 col-md-12 col-lg-12" aria-labelledby="intro_title">
-		<h1 class="intro__title" id="intro_title">Title of Enewsletter Here</h1>
-			<p class="intro__issue">FALL 2017</p>
+		<h1 class="intro__title intro__title-landingpg" id="intro_title">Title of Enewsletter Here</h1>
+			<p class="intro__issue intro__issue-landingpg">FALL 2017</p>
```

Comment:
Maybe instead of specifying the modifier as belonging to the landing page, we could just use `lg` instead. So instead of `intro__issue-landingpg`, have it be `intro_issue-lg`.